tests/library/defaultbrowsercontext-2.spec.ts

```diff
@@ -223,35 +223,3 @@ it('should connect to a browser with the default page', async ({ browserType,cre
   expect(context.pages().length).toBe(1);
   await context.close();
 });
-
-it('route.continue should delete the origin header', async ({ launchPersistent, server, isAndroid, browserName }) => {
```

Comment:
Please revert this, see corresponding bug, the origin header is not deletable in some browsers.

Reply:
It is a dupe of "should delete the origin header" in tests/page/page-request-continue.spec.ts